Install won't complete installing

Hello,

I am trying to install the windows-build-tools for days now and it does not seem to complete the install.

After some googling I read [here](https://github.com/felixrieseberg/windows-build-tools/issues/47) that I could try installing it with a different version. 

I have tried to install with the following commands:
```
npm install --global --production windows-build-tools
npm install --global --production windows-build-tools@1.3.2
npm install --global --production windows-build-tools@1.4.2
```

First, Python seem to be the problem, but after renaming the `*.amd64.msi` packages to `*.msi`, Python installed fine.

It then seems to freeze after the log line:
> Apply complete, result: 0x0, restart: None, ba requested restart:  No

It freezes after this log line with every version I tried.

### System

- Windows 10 Pro, build 16299 (1709)
- NPM 5.0.3
- Node 8.1.4

I've used an elevated PowerShell to execute the commands.

### Extra logging

```
[0ED0:07FC][2018-03-26T20:17:40]i000: MUX:  Source confirmed
[0ED0:1158][2018-03-26T20:17:40]i000: MUX:  ExecutePackageBegin for secondary installer
[0ED0:1158][2018-03-26T20:17:40]i000: MUX:  Checking to see if the secondary installer pipe should be created
[0ED0:1158][2018-03-26T20:17:40]i000: MUX:  Do not need to create the secondary installer pipe
[1DA8:13F4][2018-03-26T20:17:40]i301: Applying execute package: VSSecondaryInstaller_box, action: Install, path: C:\ProgramData\Package Cache\190F1B6CB7BCB1888AB6A3B5668008E1B51E9802\packages\SecondaryInstaller\SecondaryInstaller.exe, arguments: '"C:\ProgramData\Package Cache\190F1B6CB7BCB1888AB6A3B5668008E1B51E9802\packages\SecondaryInstaller\SecondaryInstaller.exe" /q /BurnBundleProviderKey {a9528995-e130-4501-ae19-bbfaddb779cc} /lcid 1033 /cacheLastUsedFeed "C:\Users\SomeUser\AppData\Local\Temp\-1436953092.xml" /primarylog "C:\Users\SomeUser\.windows-build-tools\build-tools-log.txt" /FeedUriToRegister "http://go.microsoft.com/fwlink/?LinkID=659004&clcid=0x409" /FeedRegistryKey "Software\Microsoft\VisualStudio\14.0\Setup\VisualCppBuildTools\Full" -burn.ancestors={a9528995-e130-4501-ae19-bbfaddb779cc}'
[0ED0:1158][2018-03-26T20:17:47]i000: MUX:  Installation size in bytes for package: VSSecondaryInstaller_box MaxAppDrive: 0  MaxSysDrive: 0  AppDrive: 0  SysDrive: 0
[0ED0:1158][2018-03-26T20:17:47]i000: MUX:  ExecutePackageComplete for secondary installer
[0ED0:1158][2018-03-26T20:17:47]i000: MUX:  Return Code:0x0 Msi Messages:0 Result Detail:0 Restart:None
[0ED0:1158][2018-03-26T20:17:47]i000: MUX:  Reset execution Result
[0ED0:1158][2018-03-26T20:17:47]i000: MUX:  Reset Result
[0ED0:1158][2018-03-26T20:17:47]i000: Setting string variable 'IsInstalled' to value '1'
[0ED0:1158][2018-03-26T20:17:47]i000: Setting string variable 'BundleResult' to value '0'
[0ED0:1158][2018-03-26T20:17:47]i000: MUX:  Engine Requested: None
[0ED0:1158][2018-03-26T20:17:47]i319: Applied execute package: VSSecondaryInstaller_box, result: 0x0, restart: None
[1DA8:13F4][2018-03-26T20:17:47]i372: Session end, registration key: SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{a9528995-e130-4501-ae19-bbfaddb779cc}, resume: ARP, restart: None, disable resume: No
[1DA8:13F4][2018-03-26T20:17:47]i371: Updating session, registration key: SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{a9528995-e130-4501-ae19-bbfaddb779cc}, resume: ARP, restart initiated: No, disable resume: No
```

### Reason for installing `windows-build-tools`

I am trying to install a `npm` package and this threw the following error:

    MSBUILD : error MSB3428: Could not load the Visual C++ component "VCBuild.exe". To fix this, 1) install the .NET Framework 2.0 SDK, 2) install Microsoft Visual Studio 2005 or 3) add the location of the component to the system path if it is installed elsewhere.


I am not even sure that the `windows-build-tools` will solve this, but after some searching, the solution should be installing `windows-build-tools`, so that's where I got stuck :)

--- a/__tests__/utils/installation-success.ts
+++ b/__tests__/utils/installation-success.ts
@@ -23,6 +23,11 @@ describe('installation-success', () => {
         isPythonSuccess: false
       });
 
+      expect(includesSuccess('Setting string variable \'IsInstalled\' to value \'1\'')).toEqual({
+        isBuildToolsSuccess: true,
+        isPythonSuccess: false
+      });
+
       expect(includesSuccess('Blubblub')).toEqual({
         isBuildToolsSuccess: false,
         isPythonSuccess: false

--- a/src/utils/installation-sucess.ts
+++ b/src/utils/installation-sucess.ts
@@ -8,7 +8,8 @@ export function includesSuccess(input: string = '') {
     // Success strings for build tools (2015)
     isBuildToolsSuccess = input.includes('Variable: IsInstalled = 1') ||
       input.includes('Variable: BuildTools_Core_Installed = ') ||
-      input.includes('WixBundleInstalled = 1');
+      input.includes('WixBundleInstalled = 1') ||
+      input.includes('Setting string variable \'IsInstalled\' to value \'1\'');
   } else {
     // Success strings for build tools (2017)
     isBuildToolsSuccess = input.includes('Closing installer. Return code: 3010.') ||

